.github/ISSUE_TEMPLATE/critical_nightly_regression.md

```diff
@@ -12,5 +12,3 @@ Troubleshooting guide: https://rust-analyzer.github.io/book/troubleshooting.html
 
 Please try to provide information which will help us to fix the issue faster. Minimal reproducible examples with few dependencies are especially lovely <3.
 -->
-
```

Comment:
Why is this removed?

Reply:
The text felt pointless to me, while just adding noise to the hopefully useful text authored by the submitter. The label itself already states this